metricbeat/module/kafka/producer/_meta/docs.asciidoc

```diff
@@ -0,0 +1,20 @@
+This metricset periodically fetches JMX metrics from Kafka Producers JMX.
+
+[float]
+=== Compatibility
+The module has been tested with Kafka 2.1.1. Other versions are expected to work.
+
+[float]
+=== Usage
+The Producer metricset requires <<metricbeat-module-jolokia,Jolokia>>to fetch JMX metrics. Refer to the link for instructions about how to use Jolokia.
```

Comment:
We should also mention something about the metricsets requiring Jolokia in the module documentation (`metricbeat/module/kafka/_meta/docs.asciidoc`).